Sources/ClientRuntime/Config/DefaultClientConfiguration.swift

```diff
@@ -31,5 +31,10 @@ public protocol DefaultClientConfiguration: ClientConfiguration {
     /// If none is provided, only a default logger provider will be used.
     var telemetryProvider: TelemetryProvider { get set }
 
+    /// Add an `InterceptorProvider` that will be used to provide interceptors for all operations.
```

Comment:
Q: Interceptor providers apply to Http operations + all other operations vs Http Interceptor providers apply to only http operations?

Reply:
That is correct